spinalcordtoolbox/deepseg/models.py

```diff
@@ -0,0 +1,94 @@
+# coding: utf-8
+"""
+Deals with models for deepseg module. Available models are listed under MODELS.
+"""
+
+
+import os
+import json
+import logging
+
+import spinalcordtoolbox as sct
+import spinalcordtoolbox.download
+
+
+logger = logging.getLogger(__name__)
+
+# List of models. The convention for model names is: (species)_(university)_(contrast)_region
+# Regions could be: sc, gm, lesion, tumor
+# TODO: add mirror
+# TODO: add a test to make sure all fields are present in the dict below
+# TODO: deal with "Folder __MACOSX"
+MODELS = {
+    't2star_sc':
+        {'url': 'https://osf.io/v9hs8/download?version=2',
+         'description': 'Cord segmentation on T2*-weighted contrast.',
+         'default': True},
+    'mice_uqueensland_sc':
+        {'url': 'https://osf.io/nu3ma/download?version=4',
+         'description': 'Cord segmentation on mouse MRI. Data from University of Queensland.',
+         'default': False},
+    'mice_uqueensland_gm':
+        {'url': 'https://osf.io/mfxwg/download?version=4',
+         'description': 'Gray matter segmentation on mouse MRI. Data from University of Queensland.',
+         'default': False},
+    }
+
+
+def folder(name_model):
+    """
+    Return absolute path of deep learning models.
+    :param name: str: Name of model.
+    :return: str: Folder to model
+    """
+    return os.path.join(sct.__models_dir__, name_model)
+
+
+def install_model(name_model):
+    """
+    Download and install specified model under SCT installation dir.
+    :param name: str: Name of model.
+    :return: None
+    """
+    logger.info("\nINSTALLING MODEL: {}".format(name_model))
+    sct.download.install_data(MODELS[name_model]['url'], os.path.split(folder(name_model))[0])
```

Comment:
This is nitpicking but:

```suggestion
    sct.download.install_data(MODELS[name_model]['url'], os.path.dirname(folder(name_model)))
```